basic-features.md

```diff
@@ -1,80 +1,136 @@
+| [Transactional `LOAD DATA`](/sql-statements/sql-statement-load-data.md)                                  | Y            | Y            | N            | N            | N            | N            |
+| [Database migration toolkit (DM)](/migration-overview.md)                                                | Y            | Y            | Y            | Y            | Y            | Y            |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)                                                      | Deprecated   | Deprecated   | Deprecated   | Y            | Y            | Y            |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md)                                                    | Y            | Y            | Y            | N            | N            | N            |
+| **Management, observability and tools**                                                                  | **5.1**      | **5.0**      | **4.0**      | **3.1**      | **3.0**      | **2.1**      |
+| [TiDB Dashboard](/dashboard/dashboard-intro.md)                                                          | Y            | Y            | Y            | N            | N            | N            |
+| [SQL diagnostics](/information-schema/information-schema-sql-diagnostics.md)                             | Experimental | Experimental | Experimental | N            | N            | N            |
+| [Information schema](/information-schema/information-schema.md)                                          | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Metrics schema](/metrics-schema.md)                                                                     | Y            | Y            | Y            | N            | N            | N            |
+| [Statements summary tables](/statement-summary-tables.md)                                                | Y            | Y            | Y            | N            | N            | N            |
+| [Slow query log](/identify-slow-queries.md)                                                              | Y            | Y            | Y            | Y            | Y            | Y            |
+| [TiUP deployment](/tiup/tiup-overview.md)                                                                | Y            | Y            | Y            | N            | N            | N            |
+| Ansible deployment                                                                                       | N            | N            | Deprecated   | Y            | Y            | Y            |
+| [Kubernetes operator](https://docs.pingcap.com/tidb-in-kubernetes/)                                      | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Built-in physical backup](/br/backup-and-restore-use-cases.md)                                          | Y            | Y            | Y            | N            | N            | N            |
```

Comment:
@bb7133 : Can you please review this section (Management, observability and tools)?

Reply:
Yes, LGTM.